scripts/ci/libraries/_initialization.sh

```diff
@@ -929,7 +929,7 @@ function initialization::ga_env() {
 function initialization::ver() {
   # convert SemVer number to comparable string (strips pre-release version)
   # shellcheck disable=SC2086,SC2183
-  printf "%04d%04d%04d%.0s" ${1//[.-]/ }
+  printf "%03d%03d%03d%.0s" ${1//[.-]/}
```

Comment:
```suggestion
  printf "%d%03d%03d%.0s" ${1//[.-]/}
```
Do we need leading zeros?

Reply:
No. We don't. But it does not hurt (as we need to handle leading zeros anyway if the major version actually is `0`).

Reply:
0.10.1 -> 0010001